modules/tinymce/src/models/dom/main/ts/table/api/Options.ts

```diff
@@ -102,6 +102,11 @@ const register = (editor: Editor): void => {
     processor: 'boolean',
     default: true
   });
+
+  registerOption('table_merge_content_on_paste', {
```

Comment:
This should also be added to the EditorOptions.ts file so that it becomes an exported type for the `tinymce.init` call.

- [ ] Add option to `EditorOptions.ts`

Reply:
From the looks of it, no option in this file is added to `EditorOptions` interface. So should they all be added to the interface or should we just not add this one as well ?

Reply:
Ahh sorry. I mean OptionTypes to the interfaces there.

Reply:
Right because they are table plugin options right but now they are part of the core so not sure really.